File: PRAK304-2210817220019-DHEA_APRILINDA_UTAMI.py
bil = int(input())
if bil==0:
    print("Nol")
elif bil>0 and bil<=10:
    print("Satuan")
elif bil>10 and bil<20:
    print("Belasan")
elif bil>=20 and bil<100:
    print("Puluhan")
elif bil>=100:
    print("Anda Menginput Melebihi Limit Bilangan")

File: PRAK304-2210817220019-DHEA_APRILINDA_UTAMI.c
#include <stdio.h>

int main()
{
    int bil;
    scanf("%d", &bil);
    if (bil>0 && bil<10)
    {
        printf("satuan");
    }
    else if (bil==0)
    {
        printf("Nol");
    }
    else if (bil>=10 && bil<20)
    {
        printf("Belasan");
    }
    else if (bil>=20 && bil<100)
    {
        printf("Puluhan");
    }
    else if (bil>=100)
    {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
}